### PR TITLE
Add tagline to ASCDB

### LINKED ISSUE #5: ASCDB needs a tagline
**Data Location**
ReactionDataset ASCDB

**Describe the issue**
Its tagline is None.

### PR DESCRIPTION
**Pull request by mattwelborn**
This PR adds a tagline to ASCDB. It also updates the `add_dataset.py` template to include tagline and description. 

Closes #5 and #7.

### CODE REVIEW

File: modifications/2020-03-12-ASCDB-tagline/add_tagline.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import qcportal as ptl
+
+client = ptl.FractalClient.from_file()
+
+ds = client.get_collection("ReactionDataset", "ASCDB")
+ds.data.__dict__["tagline"] = (
```
Comment (dgasmith) [inline modifications/2020-03-12-ASCDB-tagline/add_tagline.py]: Looking at this in this form, I think this is long enough to be a description. We should probably try to cap tagline at 100 characters or so.

Tagline: `A statistically significant subset of the MGCDB84, GMTKN55, and Minnesota 2015B databases.`

@PierMorgante

Comment (PierMorgante) [reply modifications/2020-03-12-ASCDB-tagline/add_tagline.py]: Could you please change it to "A statistically significant database generated from MGCDB84, GMTKN55, and Minnesota 2015B."?

Comment (mattwelborn) [reply modifications/2020-03-12-ASCDB-tagline/add_tagline.py]: Changes made.

Comment (PierMorgante) [reply modifications/2020-03-12-ASCDB-tagline/add_tagline.py]: Thank you!

**Review by dgasmith — APPROVED** [review]

**Comment by mattwelborn** [comment]
I will commit and merge.

**Comment by mattwelborn** [comment]
Commit done. Still need merge permissions.

**Pull request merged**